src/m2ee/config.py

```diff
@@ -89,25 +89,20 @@ def _setup_classpath(self):
 
     def _try_load_json(self, jsonfile):
         logger.debug("Loading json configuration from %s" % jsonfile)
-        fd = None
         try:
-            fd = open(jsonfile)
+            with open(jsonfile) as fd:
```

Comment:
I noticed the usage of fd for a file object earlier (wrote that myself in the past), but didn't fix it yet. As it obviously sets a wrong example, we should now.

If i see fd as variable, I expect a simple integer number which is a file descriptor, and nothing else.

Can you please s/fd/f/ in all those places?

Reply:
Done.
`ack fd -l | xargs sed -i 's/fd/f/'`
`ack close_fs -l | xargs sed -i 's/close_fs/close_fds/'`